README.md

```diff
@@ -164,8 +164,9 @@ const config = await loadConfig(configFile, cwd);
 ## Other ways to use SVGO
 
 | Method | Reference |
-| --- | --- |
-| Web app | [SVGOMG](https://jakearchibald.github.io/svgomg/) |
+| ------ | --------- |
+| Web app | [SVGOMFG](https://omfg.svg.beauty) |
```

Comment:
As I said before, we've been discussing this with the maintainer of SVGOMG, and it should be getting more regular updates soon.

See: https://github.com/svg/svgo/issues/1860#issuecomment-1832843455